src/main/java/com/spotify/docker/client/DefaultDockerClient.java

```diff
@@ -2528,6 +2526,8 @@ private String message(final Response response) {
   }
 
   private String authHeader(final RegistryAuth registryAuth) throws DockerException {
+    // the docker daemon requires that the X-Registry-Auth header is specified
+    // with a non-empty string even if your registry doesn't use authentication
```

Comment:
is there any reference to this need? It is not clear in https://docs.docker.com/engine/api/v1.30/#section/Authentication